src/extensions/exception-autocapture/error-conversion.ts

```diff
@@ -177,13 +221,7 @@ export function errorToProperties([event, source, lineno, colno, error]: ErrorEv
         $exception_level: isSeverityLevel(errorProperties.$exception_level)
             ? errorProperties.$exception_level
             : 'error',
-        ...(source
-            ? {
-                  $exception_source: source, // TODO get this from URL if not present
-              }
-            : {}),
-        ...(lineno ? { $exception_lineno: lineno } : {}),
-        ...(colno ? { $exception_colno: colno } : {}),
```

Comment:
Let's remove these from the other places they're referenced in a follow up PR

Reply:
I think they're mentioned in the new Rust service. They're also in the taxonomy descriptions but that's less important because arguably they should stick around there...

Reply:
oh it's optional, all good, need it to stay there to support old versions that might send events 💀 , but yeah we don't need to use this at all imo